src/components/phase-banner/index.md

```diff
@@ -8,9 +8,34 @@ layout: layout-pane.njk
 ---
 
 {% from "_example.njk" import example %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
 
 Use the phase banner component to show users your service is still being worked on.
 
+{% set wcagCallout %}
+
+{{ govukTag({
+  text: "WCAG 2.2",
+  classes: "app-tag"
+}) }}
+
+### New WCAG 2.2 criteria affects this component
```

Comment:
Should components with WCAG 2.2 changes related to only 1 criterion use 'criterion affects'?

And should components with changes due to multiple relevant WCAG 2.2 criteria use 'criteria affect'?
```suggestion
### New WCAG 2.2 criterion affects this component
```

Reply:
To keep it simple we'll keep it in plural – also Style guide for instances where there's both singular and plural, we go with plural. For example, say "Things" instead of "Thing" or "Thing(s)".

Will also figure out the grammar question for after tomorrow's publish at least.